src/ac/ilien

```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
-./komodo-cli -ac_name=ILN $1 $2 $3 $4 $5 $6
```

Comment:
Who do we remove it here and not in other places?

Reply:
Because there are 2 files, `ilien` & `iln`. `ilien` is just 1 file which represents ILN. And, `iln` file is still in there.

Reply:
Got it, thank you! have not noticed that there is another file for this AC